src/components/review.jsx

```diff
@@ -27,10 +30,49 @@ const Review = ({ loading, setLoading }) => {
         <h5 className="comment-header">
           {singleReview.comment_count} Comments
         </h5>
-        <button className="like-button">👍</button>
+        <button
```

Comment:
I would consider extracting this vote button functionality out into it's own component eventually. Probably a nice refactor for when the main bulk of tickets are complete...

At the moment your buttons have some strange functionality. A second click on an up or down vote will undo, which is fine, but it would be nice to give our user some notification of this: Perhaps we could change the value of the button to be 'undo' when it has been pressed?